### PR TITLE
chore: update node engines to next LTS

### PR DESCRIPTION
**Pull request by nextcloud-command**
This PR was automatically generated by the engines dispatcher.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
     "extends @nextcloud/browserslist-config"
   ],
   "engines": {
-    "node": "^16.0.0",
-    "npm": "^7.0.0 || ^8.0.0"
+    "node": "^20.0.0",
```
Comment (szaimen) [inline package.json:37]: @skjnldsv I just saw 20 is not LTS yet? So why upgrade already?

Comment (skjnldsv) [reply package.json:37]: Because this is for nc 28, and LTS will be Node 20 when we relelase

**Review by skjnldsv — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (szaimen) [reply package.json:37]: Ah I see

**Review by danxuliu — COMMENTED** [review]
UI is broken after this update, although I have not checked exactly why.

The PDF viewer should be fixed to use the new node engines, or the older engines should be used again.

**Comment by szaimen** [comment]
I would suggest using the older node version again then

**Comment by danxuliu** [comment]
> I would suggest using the older node version again then

Fine by me: https://github.com/nextcloud/files_pdfviewer/pull/776

**Pull request merged**